packages/calypso-e2e/src/lib/pages/p2-page.ts

```diff
@@ -0,0 +1,28 @@
+import { Page } from 'playwright';
+
+const selectors = {
+	inputPrompt: 'span.p2020-editor-placeholder__prompt',
+};
+
+/**
+ *
+ */
+export class P2Page {
+	private page: Page;
+
+	/**
+	 * Constructs an instance of the component.
+	 *
+	 * @param {Page} page The underlying page.
+	 */
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	/**
+	 *
+	 */
+	async focusInlineEditor(): Promise< void > {
+		await this.page.click( selectors.inputPrompt );
```

Comment:
I've treated this method as being separate from the `IsolatedBlockEditorComponent` due to the primary highlight being on the P2 page itself when first landing on a P2 site.